src/vos/tests/csum_extent_tests.c

```diff
@@ -661,13 +661,19 @@ write_to_extent(struct extent_key *extent_key, daos_recx_t *extent,
 static uint8_t *
 allocate_random(size_t len)
 {
-	uint8_t *result = (uint8_t *)malloc(len);
+	uint8_t *result;
 
-	D_ASSERT(result != NULL);
-	dts_buf_render((char *) result, (unsigned int) len);
+	D_ALLOC(result, len);
+	if (result)
+		dts_buf_render((char *) result, (unsigned int) len);
 	return result;
 }
 
+#define	D_ALLOC_RAND(ptr, len) do { \
```

Comment:
It would be more consistent to put do on next line and indent the body of the do while as if it were.  Though it's unclear why we can't simply make this a function.   Also D_ is a gurt prefix.   We can just use ALLOC_RAND here

Reply:
sure, I'll make that change.